src/game/hud.cpp

```diff
@@ -1355,13 +1383,33 @@ namespace hud
                 }
                 if(crosshairhitspeed && totalmillis-game::focus->lasthit <= crosshairhitspeed)
                 {
-                    vec c2(1, 1, 1);
-                    if(hitcrosshairtone) skewcolour(c2.r, c2.g, c2.b, hitcrosshairtone);
-                    else c2 = c;
-                    drawpointertex(getpointer(POINTER_HIT, game::focus->weapselect), cx-cs/2, cy-cs/2, cs, c2.r, c2.g, c2.b, crosshairblend*hudblend);
+                    hit = true; 
+                    if (hitcrosshairtone)
+                    {
+                        skewcolour(color_hit.r, color_hit.g, color_hit.b, hitcrosshairtone);
```

Comment:
What exactly does this line do?

Reply:
you mean the skewcolor thing?

Reply:
I assume you mean that, and tbh, I'm not entirely sure but I assume it makes the crosshair pulsate in these kinda-redish colors. It was there before so I just left it there

Reply:
Yep. I'm curious. A comment might be appropriate.